### PR TITLE
Fixed parsing issues

### PR DESCRIPTION
**Pull request by VenkataChalla554**
## Context
Some of logs  for  deployment ingress-nginx-controller   not parsing , due to parsing pattern not matching

## Changes proposed in this pull request
Adding matching expression for multiple patterns
## Guidance to review
 Verify logs on logit.io where logs start with ip adress and log level , it should parse properly.
## Before merging
 NA

## After merging
Need to apply into all the environments.

## Checklist

- [x] I have performed a self-review of my code, including formatting and typos
- [x] I have [cleaned the commit history](https://www.annashipman.co.uk/jfdi/good-pull-requests.html)
- [x] I have added the `Devops` label
- [x] I have attached the pull request to the trello card

### CODE REVIEW

**Comment by VenkataChalla554** [comment]
[logit - fix ingress grok failures](https://trello.com/c/DNcgbmow/1912-logit-fix-ingress-grok-failures)

File: documentation/logit-io.md [hunk 1]
```diff
@@ -67,7 +67,9 @@ filter {
...
+        match => { "message" => [
+          "%{IPORHOST:[source][ip]} - (?:%{DATA:[url][username]}|-)? \[%{HTTPDATE:[ingress][time]}\] \"%{WORD:[http][request][method]} %{DATA:[url][original]} HTTP/%{NUMBER:[http][version]}\" %{NUMBER:[http][response][status_code]} %{NUMBER:[http][response][body][bytes]} \"%{DATA:[http][request][referrer]}\" \"%{DATA:[ingress][agent]}\" %{NUMBER:[http][request][bytes]} %{NUMBER:[ingress][request_time]} \[%{DATA:[ingress][proxy][upstream][name]}\] \[%{DATA:[ingress][proxy][alternative_upstream_name]}\] (?:%{IP:[ingress][upstream][addr]}:%{NUMBER:[ingress][upstream][port]}|-) (?:%{NUMBER:[ingress][upstream][response][length]}|-) (?:%{NUMBER:[ingress][upstream][response][time]}|-) (?:%{NUMBER:[ingress][upstream][status]}|-) %{NOTSPACE:[http][request][id]}"
```
Comment (saliceti) [inline documentation/logit-io.md:71]: As discussed, the change is not required as we will ignore the rare exceptions

File: documentation/logit-io.md [hunk 2]
```diff
@@ -213,3 +215,6 @@ Logs collected by filebeat are stored in daily index `filebeat-<date>`. Other in
 1. List indices: `GET /_cat/indices/`
 1. Delete opensearch-sap-log-types-config index: `DELETE /.opensearch-sap-log-types-config`
 1. Delete logstash indices: `DELETE /logstash*`
+
+## Debug Gork Expression
```
Comment (saliceti) [inline documentation/logit-io.md:219]: Typo: Grok

File: documentation/logit-io.md [hunk 3]
```diff
@@ -213,3 +215,6 @@ Logs collected by filebeat are stored in daily index `filebeat-<date>`. Other in
...
+The following tool is useful for debugging Grok expressions:  https://grokconstructor.appspot.com/
```
Comment (saliceti) [inline documentation/logit-io.md:220]: This could be moved under the ## Logstash inputs section instead

**Review by saliceti — APPROVED** [review]

**Pull request merged**